### DISCUSSION TITLE
How to Customize 404 Pages

### DISCUSSION THREAD

**Discussion by JiaZhiheng** [nuxt/nuxt#16449] [Questions] [answered]
如何自定义404页面呢

**Comment by TheAlexLichter** [comment, marked as answer]
Hey! See https://v3.nuxtjs.org/getting-started/error-handling/#rendering-an-error-page - `error.vue` in the project root.

**Reply by JiaZhiheng** [reply]
Your answer solved my problem，Thank you very much !

**Reply by typed-sigterm** [reply]
Will using `[...slug].vue` to handle 404 affect SEO?
